### PR TITLE
APCV-1227- Update changelog for CS

### PR DESCRIPTION
**Pull request by palatsangeetha**
This PR has the changes for the issue - [APCV-1227](https://dcsa.atlassian.net/jira/software/c/projects/APCV/boards/47/backlog?selectedIssue=APCV-1227)

[APCV-1227]: https://dcsa.atlassian.net/browse/APCV-1227?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: cs/v1/README.md [hunk 1]
```diff
@@ -1,8 +1,98 @@
 # Commercial Schedule (CS) API
 
+The DCSA Commercial Schedules API is specified on [**SwaggerHub**](https://app.swaggerhub.com/apis/dcsaorg/DCSA_CS).
 
+### <a name="v100B1"></a>[Release v1.0.0 (24 September 2024)](https://app.swaggerhub.com/apis-docs/dcsaorg/DCSA_CS/1.0.0)
```
Comment (HenrikHL) [inline cs/v1/README.md:5]:
```suggestion
### <a name="v100"></a>[Release v1.0.0 (24 September 2024)](https://app.swaggerhub.com/apis-docs/dcsaorg/DCSA_CS/1.0.0)
```

Comment (HenrikHL) [reply cs/v1/README.md:5]: I assume it is the final ;) 100B1 refers to 100 Beta 1 ;-)

Comment (HenrikHL) [reply cs/v1/README.md:5]: As a FYI - this also needs to be updated in the YAML file so it links forth and back

File: cs/v1/README.md [hunk 2]
```diff
@@ -1,8 +1,98 @@
...
+    - `departureDateTime` changed to `departureStartDate`,`departureEndDate`
+    - `arrivalDateTime` changed to `arrivalStartDate`,`arrivalEndDate`
+    - `isTranshipment` changed to `maxTranshipment`($int32)
+    - Added `cursor`
+  - Response
+    - Added `receiptTypeAtOrigin`,`receiptTypeAtDestination`,`cutOffTimes`,`solutionNumber` to the response
+    - `transitTime` format changed from $int64 to $int32
+    - `facilitySMDGCode` split to `facilityCode` and `facilityCodeListProvider`(BIC,SMDG)
+    - `dateTime` to be expressed in local time with offset (YYYY-MM-DDTHH:MM:SS±HH:MM)
+      - with exception of errorDateTime which is in UTC without offset.
```
Comment (HenrikHL) [inline cs/v1/README.md:27]:
```suggestion
      - with exception of `errorDateTime` which is in UTC without offset.
```

File: cs/v1/README.md [hunk 3]
```diff
@@ -1,8 +1,98 @@
...
+    - location object changed from discriminators for each location to an object with types of different locations as optional properties
+    - Added `cutOffTimes`
+    - Added `servicePartners` object (mandatory) and moved properties `carrierCode`,`carrierCodeListProvider`,`carrierServiceName`,`carrierServiceCode`,`carrierImportVoyageNumber`,`carrierExportVoyageNumber` to it
+    - Added `vessel` object and moved `vesselIMONumber`,`name`(before `vesselName`),`operatorCarrierCode`,`operatorCarrierCodeListProvider` to it.
+    - Added `MMSINumber`,`flag`,`callsign` to `vessel` object
+    - Changed `vesselOperatorSMDGLinerCode` to `operatorCarrierCode` + `operatorCarrierCodeListProvider`
+    - Changed `vesselIMONumber` length from String of 7 to String minimum length 7 and maximum length 8
+    - Changed maximum length of vessel name from 35 to 50
+    - Changed Timestamp array attribute name from `timestamp` to `timestamps`
+    - Added `X-Next-Page-Cursor` to the response header for pagination
```
Comment (HenrikHL) [inline cs/v1/README.md:68]: Why is it called X-Next-Page-Cursor here when it is called Next-Page-Cursor in the endPoint described above?

File: cs/v1/README.md [hunk 4]
```diff
@@ -1,8 +1,98 @@
...
+- locations are handled differently going forward. Locations are no longer defined using a `oneOf` but rather via optional properties.
+- Added Pagination to all the endpoints
 
+## All changes
+- Point to Point
+  - Filters
+    - `departureDateTime` changed to `departureStartDate`,`departureEndDate`
+    - `arrivalDateTime` changed to `arrivalStartDate`,`arrivalEndDate`
+    - `isTranshipment` changed to `maxTranshipment`($int32)
+    - Added `cursor`
```
Comment (HenrikHL) [inline cs/v1/README.md:21]: Was limit already added to this endPoint since you don't mention it?

Comment (palatsangeetha) [reply cs/v1/README.md:21]: yes 'limit' for point to point was already there in the Beta version

File: cs/v1/README.md (hunk 1 above)
Comment (palatsangeetha) [reply cs/v1/README.md:5]: Thanks Henrik for pointing that out. Updated the API as well

**Review by HenrikHL — APPROVED** [review]

**Pull request merged**